lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Migrations.hs

```diff
@@ -513,9 +518,10 @@ spec = describe "SHELLEY_MIGRATIONS" $ do
                     <$> listUnusedAddresses @n ctx sourceWallet
                 liftIO $ _mintSeaHorseAssets ctx
                     perEntryAssetCount
+                    batchSize
                     perEntryAdaQuantity
                     sourceAddresses
-                waitForTxImmutability ctx
+            waitForTxImmutability ctx
```

Comment:
@Anviking Is this safe? If we don't wait for tx immutability between each batch, then how can we be sure we're getting a batch of unused addresses when we call `listUnusedAddresses`?